sys-apps/ignition/ignition-9999.ebuild

```diff
@@ -12,7 +12,7 @@ inherit coreos-go cros-workon systemd udev
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm64"
 else
-	CROS_WORKON_COMMIT="d5545707b879e507749efed89881f987bd0ec81e" # main
+	CROS_WORKON_COMMIT="fef845a7d5ba463b4a93a5ed03a76721164c4a84" # v2.14.0
```

Comment:
While at it - the `COREOS_GO_PACKAGE` variable above says `github.com/flatcar-linux/ignition/v2`. Should it say `coreos` rather?

Not sure that it matter, though - building the package worked so far anyway. But we should consider updating the ebuild to stop using `cros-workon` and `coreos-go` eclasses, I think. But not in this PR. Let's keep the scope of the PR limited to ignition update.